bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1980,10 +1980,19 @@ void ReplicaImp::onNewView(const std::vector<PrePrepareMsg *> &prePreparesForNew
     primaryLastUsedSeqNum = lastStableSeqNum;
     strictLowerBoundOfSeqNums = lastStableSeqNum;
     maxSeqNumTransferredFromPrevViews = lastStableSeqNum;
+    LOG_INFO(GL,
+             "No PrePrepare-s for the new view: " << KVLOG(
+                 primaryLastUsedSeqNum, strictLowerBoundOfSeqNums, maxSeqNumTransferredFromPrevViews));
```

Comment:
Might be we should add "***" prefix, to be consistent with the other View Change associated logs.